main.cpp

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include <vector>
+#include "unit.h"
+
+using namespace std;
+
+void printStatus(vector<unit*> alive){
+    for( auto v:  alive)
+        cout << v->getName() << ": HP: " << v->getHp() << ", DMG: " << v->getDmg() << endl;
+}
+
+void printAttack(string n1, string n2){
```

Comment:
a string nem olyan nagy, de az is egy osszetett adattipus, amit itt tok feleslegesen masolunk. Illetve const-ok?

Reply:
Teljesen jogos. Javítva.